sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgInstructionFactory.cs

```diff
@@ -450,6 +450,14 @@ private IEnumerable<Instruction> ProcessBaseMethodDeclaration(BaseMethodDeclarat
         private IEnumerable<Instruction> ProcessMemberAccessExpression(MemberAccessExpressionSyntax memberAccessExpression)
         {
             var memberAccessSymbol = GetSymbol(memberAccessExpression);
```

Comment:
Maybe that would be a wrong suggestion but maybe we could update the `GetSymbol` method to include the test of the symbol kind and return null if that's not a kind of symbol we are interested into.

Reply:
Something like _GetSymbol<T>()_? It seems like this is a fairly common pattern, so that would be neater.

Reply:
Yes. The body of `GetSymbol` would actually retrieve the symbol and then test at the kind of the symbol. If symbol kind is contained in the set of unsupported kinds we return null, otherwise we return the symbol.